governance.md

```diff
@@ -153,49 +289,135 @@ are enumerated below.
 
 ## Other duty-specific github roles:
 - [**Github Org Owner**](https://github.com/orgs/kubernetes/people?utf8=%E2%9C%93&query=%20role%3Aowner):
-  can create repos, do ~any github action; the number of
+  can create repos, and do any github action; the number of
```

Comment:
And we need people in multiple continents for timezone coverage -- it's an oncall function, too.

Add this to the umbrella issue.